app/views/MaintainThisTrustView.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.FrontendAppConfig
+
+@this(
+    main_template: MainTemplate,
+    config: FrontendAppConfig
+)
+
+@(utr: String)(implicit request: Request[_], messages: Messages)
+
+@main_template(
+    title = messages("maintainThisTrust.title")
+) {
+
+    @components.back_link()
+
+    <span class="govuk-caption-xl">@messages("maintainThisTrust.subheading", utr)</span>
+    @components.heading("maintainThisTrust.heading")
+
+    <p>@messages("maintainThisTrust.p1.a") <strong>@utr</strong> @messages("maintainThisTrust.p1.b")</p>
```

Comment:
Really picky but according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong this would better be a <b> element, can we alternatively do it with a span and a css class to add bold?